BridgeAppSDK/SBAScheduledActivityManager.swift

```diff
@@ -382,13 +397,54 @@ public class SBAScheduledActivityManager: NSObject, SBASharedInfoController, ORK
     public func sendUpdatedScheduledActivities(scheduledActivities: [SBBScheduledActivity]) {
         SBAUserBridgeManager.updateScheduledActivities(scheduledActivities)
     }
-    
-    public func archiveResults(schedule: SBBScheduledActivity, taskViewController: ORKTaskViewController) {
+        
+    public func archiveResults(schedule: SBBScheduledActivity, taskViewController: ORKTaskViewController) -> [SBADataArchive]? {
         
         let results = activityResultsForSchedule(schedule, taskViewController: taskViewController)
         print(results)
         
-        // TODO: implement syoung 04/27/2016 Stubbed out accessor
+        var archives = [SBADataArchive]()
+        for activityResult in results {
+            if let activityResultResults = activityResult.results as? [ORKStepResult] {
+                let archive = SBADataArchive.init(reference: activityResult.schemaIdentifier, schemaRevision: activityResult.schemaRevision)
```

Comment:
remove the `init`. Also, see my comment above about using set method to add to userInfo dictionary. This will allow surveys to also add info to the userInfo without the SBADataArchive needing to care about those kinds of details.